File: Exercism Codes/C#/ResistorColorDuo.cs
using System;

public static class ResistorColorDuo
{
    private static string[] colorsList = new string[]
    {
        "black", "brown", "red", "orange", "yellow", "green", "blue", "violet", "grey", "white"
    };

    public static int Value(string[] colors)
    {
        return int.Parse($"{Array.IndexOf(colorsList, colors[0])}{Array.IndexOf(colorsList, colors[1])}");
    }
}

File: Exercism Codes/Kotlin/ResistorColorDuo.kt
object ResistorColorDuo {
    private val COLORS = listOf(
        Color.BLACK,
        Color.BROWN,
        Color.RED,
        Color.ORANGE,
        Color.YELLOW,
        Color.GREEN,
        Color.BLUE,
        Color.VIOLET,
        Color.GREY,
        Color.WHITE
    )
    fun value(vararg colors: Color): Int {
        return "${COLORS.indexOf(colors[0])}${COLORS.indexOf(colors[1])}".toInt()
    }
}